source/includes/_values.md

```diff
@@ -0,0 +1,49 @@
+##Passing by Value vs. Passing by Reference
+
+#Passing by Reference
+You're in an university class, given a topic on __Passing by Reference__. Your
+whole class is really struggling with the paper and everyone is complaining,
+wanting a topic change. But unfortunately, Professor JavaScript sees the value
+in the topic and it will never change. So consider this, even though the
+topic, or value, never changes, the papers handed in can have different
+contents but the topic never changes. In essence, now when talking about
+__Passing by Reference__ all those papers can be handed in no matter the
+attributes of those papers.
+
+```js
+var paperTopic = [paper1, paper2, paper3, paper4];
+
+Values in the array can be changed,
```

Comment:
If you're going to put this in the code block, make sure that you make it as comments, so the JS is still valid.